core/integration/steps/the_markets.go

```diff
@@ -103,6 +103,7 @@ func TheMarkets(
 		if err := executionEngine.SubmitMarket(context.Background(), &markets[i], "proposerID", now); err != nil {
 			return nil, fmt.Errorf("couldn't submit market(%s): %v", markets[i].ID, err)
 		}
+		// only start opening auction if the market is explicitly marked to leave opening auction now
```

Comment:
Is this comment correct? Not sure what "is passed" does, but seems like it's maybe to do with governance vote passing?